tcms/templates/include/tp_toolbar.html

```diff
@@ -0,0 +1,135 @@
+{% load i18n %}
+<style>
+    {#todo use css file#}
```

Comment:
Indeed move this to a separate css file.

Reply:
Why do we need these styles here anyway ?

Reply:
![with_styles](https://user-images.githubusercontent.com/8727922/88849476-9cbacc80-d1f2-11ea-9bb1-10ec1a69f1e6.png)
![without_styles](https://user-images.githubusercontent.com/8727922/88849480-9d536300-d1f2-11ea-8e90-4b5fc08cf4a9.png)

Explanation for what I did with the styles.

Reply:
Hmm, this doesn't look right to me. You should be able to create a toolbar and add components inside of it without needing extra CSS. The kiwi-toolbar class should be a pf-toolbar or something like that.

Reply:
See "row toolbar" and the surrounding HTML structure here https://www.patternfly.org/v3/pattern-library/forms-and-controls/toolbar/#code